install/connections.csv

```diff
@@ -845,7 +845,9 @@ TEU,10,Te Uri,Short Period,,1990-02-20T02:23:00Z,1990-05-17T17:11:00Z
 TGRZ,10,Welcome Bay Road,,,2003-02-18T08:00:01Z,9999-01-01T00:00:00Z
 THLX,10,Tinakori Hill,,0,2014-08-25T00:05:00Z,2015-03-24T00:00:00Z
 THLX,20,Tinakori Hill,,3,2014-08-25T00:05:00Z,2015-03-24T00:00:00Z
-THQ2,10,Tauhara North,,,2012-12-18T01:00:00Z,9999-01-01T00:00:00Z
+THQ2,10,Tauhara East,,,2023-08-11T02:44:00Z,2023-08-13T21:14:53Z
```

Comment:
I think we don't need this entry as that is what requires a cleanup/rename in the archive? @mnaguit Thoughts?

Reply:
We are closing LC 10? if yes, then we need a new entry for LC 11?

Reply:
Extra entry removed